src/server/v2.0/handler/base.go

```diff
@@ -101,8 +102,8 @@ func (b *BaseAPI) RequireProjectAccess(ctx context.Context, projectIDOrName inte
 	return errors.ForbiddenError(nil)
 }
 
-// RequireSysAdmin checks the system admin permission according to the security context
-func (b *BaseAPI) RequireSysAdmin(ctx context.Context) error {
+// RequireSysAdminAccess checks the system admin permission according to the security context
```

Comment:
I think to make the model more consistent we should treat sys admin equally.  i.e. check the permission regardless if the secCtx is sysadmin or not.

So the func should be `RequireAccess` and `if !secCtx.IsSysAdmin() {` will be redundant?

Could you please also consider to refactor the `HasPermission` and `HasProjectPermission` to make things more consistent and simplified.

Reply:
resolved.